.github/workflows/reusable-module-prep-release.yml

```diff
@@ -13,12 +13,22 @@ on:
       module-branch:
         description: The branch of the module to use.
         type: string
-        required: true
+        default: 'main'
+        required: false
       level:
         description: The level of release to be used.
         type: string
-        default: 'patch'
         required: true
+      json-file:
+        description: The JSON file to be used.
```

Comment:
Could we be more specific here? Used for what?

Reply:
Good point, I've updated